src/commands/docs/internal/getDotFileContent.ts

```diff
@@ -42,7 +42,7 @@ export async function getDotFileContent(
 
       fileNodes.set(fileName, {
         edges: fileOutputs,
-        label: `${fileName} | ${fileBrief}`
+        label: `${fileName} | ${fileBrief}`.replace(/"/g, '\\"')
```

Comment:
maybe we could also escape backslash here also?

Reply:
added